### DISCUSSION TITLE
iOS: sandboxed Unix/POSIX? Never got iOS because of requirements to jailbreak just to have terminal access

### DISCUSSION THREAD

**Discussion by ETERNALBLUEbullrun** [ish-app/ish#2382] [Ideas]
How to run this on new stock iOS?
The one advantage of iOS (versus Android OS) is how secure this is,
but if you have to jailbreak just to use it as a computer it has no advantages.

**Comment by saagarjha** [comment]
You can download it from the App Store here: https://apps.apple.com/us/app/ish-shell/id1436902243
